src/main/java/com/beust/jcommander/JCommander.java

```diff
@@ -1420,6 +1420,102 @@ public void addCommand(String name, Object object, String... aliases) {
         }
     }
 
+    /**
+     * This method can create a subcommand for one command.
+     * The used command like this: programName command subcommand --flag
+     * Actually this method is wrap "addCommand", add a command for one command but the main command
+     * @param name the name for subcommand
+     * @param object the class you want to create for subcommand
+     * @param parentCommand the instance of command that you want to add subcommand
+     * @param aliases the aliases for the subcommand
+     **/
+    public void addSubcommand(String name, Object object, JCommander parentCommand, String... aliases){
```

Comment:
I do not understand the benefit of this command over the existing possibility of simply calling `parentCommand.addCommand()` directly.

Reply:
@Asriter Unfortunately you have no responded to my question, so I hope you do not feel discouraged from setting the state of this PR to draft.

Reply:
@Asriter Kindly requsting your response! :-)